### PR TITLE
Jennifer Kurtto and Savannah Hayes - Medibot

### PR DESCRIPTION
**Pull request by savannah-hayes**
Jennifer and I worked on this project together. 
[https://optimistic-edison-9024ce.netlify.app/](https://optimistic-edison-9024ce.netlify.app/)

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    "Do you have a runny or stuffy nose?"
+  ]
+let psychologicalSymptoms =
+  [
+    "Are you feeling sad or depressed?",
+    "Are you unable to cope with daily problems or stress?",
+    "Are you experiencing anxiety?"
+  ]
+
+if (hours < 12) greet = "Good Morning";
```
Comment (rawisou) [inline assets/script.js:30]: Nice touch! :)

File: assets/script.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        } else {
+          practitioner = "Psychologist";
+          const sad = () => showMessage(psychologicalSymptoms[0], "bot", timeStamp());
+          setTimeout(sad, 2000);
+          setTimeout(yesNoButtons, 2000);
+        }
+      break;
+
+      case 2:
+        showMessage(`${button.innerHTML}`, "user", timeStamp());
```
Comment (rawisou) [inline assets/script.js:189]: Great use of the _switch statement_.

I am wondering if these can also be written using nested conditional statement? Anyway, I have learned a lot from your code!

**Review by rawisou — COMMENTED** [review]
Your project has fulfilled the general requirements, blue, red and black requirements. Great job! You have refactored the code and used advanced level of functions.

Your code is clean and easy to follow.
I'm impressed and inspired by your work!

/R

**Comment by bostrombundock** [comment]
Your job is very well organized and looks tidy, which makes it easy to follow. I don´t have anything else to comment other than this is inspiring and I learned a lot.

**Pull request closed without merging**